modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -70,21 +76,34 @@ def validate_params
         validate_type_and_services_known unless params[:type].nil?
       end
 
-      def get_query_method(params)
+      def get_lat_lng(params)
         obs_fields = params.keys.map(&:to_sym)
         location_type = REQUIRED_PARAMS.find do |loc_type, req_field_names|
           no_missing_fields = (req_field_names - obs_fields).empty?
           break loc_type if no_missing_fields
         end
-        QUERY_INFO[location_type]
+
+        if location_type.eql? :address
+          GeocodingService.new.query(params[:street_address], params[:city], params[:state], params[:zip])
+        else
+          params.slice(:lat, :lng)
+        end
```

Comment:
Explicit in what way? This reads okay to me (as another outside party— @whuang85 put that block together). If there's an address, it geocodes and returns. Otherwise it returns the passed in lat/long. Rubocop favors the implicit returns, though as someone who came to Ruby from other languages, sometimes that throws me, too.

Reply:
Yeah, a final line in the method that returns a variable assigned in the if/else was what I was referring to. Obviously we don't literally need it, just trying to get a temperature check on whether folks thought it would make it easier to read.

Reply:
not sure what ruby conventions are.. or even coding best practices for VA.. Originally, I had explicit returns in each conditional and rubocop removed the `return`s

Reply:
Calling this six/half-dozen and leaving as is. Strong objections welcomed and I'll reopen.